package-res/resources/web/pentaho/type/value.js

```diff
@@ -231,6 +231,187 @@ define([
         },
         //endregion
 
+        //region creation
+        //@override
+        /**
+         * Creates an instance of this type, given an instance specification.
+         *
+         * If the specified instance specification contains an inline type reference,
+         * in property `"_"`, the referenced type is used to create the instance,
+         * as long as it is a subtype of this type. Otherwise, if it is not a subtype,
+         * an error is thrown.
+         *
+         * If the specified instance specification does not contain an inline type reference
+         * the type is assumed to be this type.
+         *
+         * An error is thrown if the type ultimately used to create the instance
+         * is an [abstract]{@link pentaho.type.Value.Meta#abstract} type.
+         *
+         * @example
+         * <caption>
+         *   Create a complex instance from a specification that carries inline type metadata.
+         * </caption>
+         *
+         * require(["pentaho/type/Context"], function(Context) {
+         *
+         *   var context = new Context({container: "data-explorer-101"});
+         *   var Value   = context.get("value");
+         *
+         *   var product = Value.meta.create({
+         *         _: {
+         *           props: ["id", "name", {name: "price", type: "number"}]
+         *         },
+         *
+         *         id:    "mpma",
+         *         name:  "Principia Mathematica",
+         *         price: 1200
+         *       });
+         *
+         *   // ...
+         *
+         * });
+         *
+         * @example
+         * <caption>
+         *   Create a list instance from a specification that carries inline type metadata.
+         * </caption>
+         *
+         * require(["pentaho/type/Context"], function(Context) {
+         *
+         *   var context = new Context({container: "data-explorer-101"});
+         *   var Value   = context.get("value");
+         *
+         *   var productList = Value.meta.create({
+         *         _: [{
+         *           props: ["id", "name", {name: "price", type: "number"}]
+         *         }],
+         *
+         *         d: [
+         *           {id: "mpma", name: "Principia Mathematica", price: 1200},
```

Comment:
I thought you meant Newton's book "Philosophiæ Naturalis Principia Mathematica", which would have been priceless, but from the context it seems you're referencing Whitehead's and Russel's book.